x-pack/legacy/plugins/code/server/lsp/language_servers.ts

```diff
@@ -69,8 +69,8 @@ export const CTAGS: LanguageServerDefinition = {
   embedPath: require.resolve('@elastic/ctags-langserver/lib/cli.js'),
   priority: 1,
 };
-export const LanguageServers: LanguageServerDefinition[] = [TYPESCRIPT, JAVA, GO];
-export const LanguageServersDeveloping: LanguageServerDefinition[] = [CTAGS];
+export const LanguageServers: LanguageServerDefinition[] = [TYPESCRIPT, JAVA, GO, CTAGS];
```

Comment:
this is only to going to be there in the `master` branch, right?

Reply:
Both `master` and `7.x`, I think. As long as the security issue is resolved, the ctags langserver can be enabled again.